examples/react-redux/src/components/footer.jsx

```diff
@@ -0,0 +1,45 @@
+import { Component } from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+
+import { SHOW_ALL, SHOW_COMPLETED, SHOW_ACTIVE } from "../constants/todo-filters";
+
+export default class Footer extends Component {
```

Comment:
Since the more recent function components are more used and are recommended over class components by the react core team shouldn't we be using them instead of class components? I am aware that this is a big change so it might make more sense to just merge this now and update it later on. 
Would love to know your opinion regarding this

Reply:
For this build, we chose a traditional class pattern to implement the todoMVC application. It's still used widely and I believe it still represents a pattern that is useful to the community. 

There will be a React specific version that will use functional components and hooks to represent a more modern approach.